### DISCUSSION TITLE
How to use / or # in attributify mode ?

### DISCUSSION THREAD

**Discussion by s3xysteak** [unocss/unocss#3325] [Q&A] [answered]
```html
//It worked
<div class="bg-pink-5/50">

//Error, because of /
<div bg-pink-5/50>

//Error, because of #
<div bg-#fff>
```

**Comment by chu121su12** [comment]
You may just want to use class since html may [not support those characters](https://www.w3.org/TR/xml/#sec-common-syn) used in those way.

**Comment by vabatta** [comment, marked as answer]
It is explained here: https://unocss.dev/transformers/attributify-jsx#caveats
